penguin/core.py

```
@@ -1828,6 +1829,73 @@ async def _emit_opencode_session_status(
+                active_requests = getattr(self, "_opencode_active_requests", None)
+                active_count = (
+                    active_requests.get(session_id, 0)
+                    if isinstance(active_requests, dict)
+                    else 0
+                )
+                if active_count <= 0:
+                    return
+                await self._emit_opencode_session_status(session_id, "busy")
+        except asyncio.CancelledError:
+            raise
+        except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)